### DISCUSSION TITLE
Welcome to ecommerce-gin-clean-arch Discussions!

### DISCUSSION THREAD

**Discussion by nikhilnarayanan623** [nikhilnarayanan623/ecommerce-gin-clean-arch#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abinth11** [comment]
Hello nikhil, amazing work. If the project is active could you please share the domain
